src/cmake/GeosxConfig.cmake

```diff
@@ -67,6 +67,10 @@ function( make_full_config_file
     set( GEOSX_LOCALINDEX_TYPE_FLAG "3" )
     set( GEOSX_GLOBALINDEX_TYPE "long long int" )
     set( GEOSX_GLOBALINDEX_TYPE_FLAG "2" )
+    set( GEOSX_LA_INTERFACE "Hypre" )
+    set( GEOSX_LA_INTERFACE_HYPRE ON )
+    set( GEOSX_LA_INTERFACE_TRILINOS OFF )
+    set( GEOSX_LA_INTERFACE_PETSC OFF )
```

Comment:
They are, but they're overridden by user host-config. One of the recent PRs actually committed changes to `doxygen/GeosxConfig.hpp`, reverting it to `Trilinos` once again, because we were missing enforcement of these LAI parameters here.